sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -573,6 +637,71 @@ case class CatalogTable(
     map
   }
 
+  def toJsonLinkedHashMap: mutable.LinkedHashMap[String, String] = {
+    val map = new mutable.LinkedHashMap[String, String]()
+
+    val filteredTableProperties = SQLConf.get
+      .redactOptions(properties.filter { case (k, v) =>
+        !k.startsWith(VIEW_PREFIX) && v.nonEmpty
+      })
+
+    val tableProperties: String =
+      if (filteredTableProperties.isEmpty) ""
+      else filteredTableProperties.toSeq
+        .sortBy(_._1)
+        .map { case (k, v) => s""""$k": "$v"""" }
+        .mkString("{", ", ", "}")
+
+    val partitionColumns = partitionColumnNames
+      .map(s => s""""$s"""")
+      .mkString("[", ", ", "]")
+
+    val lastAccess =
+      if (lastAccessTime <= 0) "\"UNKNOWN\""
+      else s""""${new Date(lastAccessTime).toInstant.toString}""""
+
+    identifier.catalog.foreach(catalog => map.put("Catalog", s""""$catalog""""))
+    identifier.database.foreach(database => map.put("Database", s""""$database""""))
+    map.put("Table", s""""${identifier.table}"""")
+    if (owner != null && owner.nonEmpty) map.put("Owner", s""""$owner"""")
+    map.put("Created Time", s""""${new Date(createTime).toInstant.toString}"""")
```

Comment:
See `microsToInstant` if you need to construct `Instant` from milliseconds (see also `millisToMicros()`)